### PR TITLE
chore: latest-rc 7.92.0 @W-0@

### PR DESCRIPTION
**Pull request by peternhale**
[skip-validate-pr]

### What does this PR do?
release latest-rc 7.92.0

Removed the old resources, the old scripts, use the new oclif version, cleaned up the package.json deps and scripts, and cleaned up the circleci config including adding windows unit test run.

Cleaning up unused files in the archives still doesn't work.  I added it to the pre and post pack:tarball but it doesn't work right in oclif. It will probably require a change there. I put XX so it doesn't work on pre and post.

### What issues does this PR fix or reference?
NA

### CODE REVIEW

**Review by amphro — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -142,67 +143,46 @@
...
-    "promote": "oclif promote",
-    "clean": "yarn clean-dist && yarn clean-test",
-    "clean-all": "scripts/clean/all",
-    "clean-build": "scripts/clean/build",
-    "clean-compile": "yarn clean-all && yarn install --pure-lockfile",
-    "clean-dist": "scripts/clean/dist",
-    "clean-test": "scripts/clean/test",
-    "compile": "scripts/build/compile",
-    "coverage-report": "scripts/test/coverage",
+    "XXpostpack:tarballs": "./scripts/verify-tarballs",
```
Comment (WillieRuemmele) [inline package.json:158]: Why is there the `XX`?

Comment (amphro) [reply package.json:158]: It's not working =( So I don't want it to run right now.

Comment (amphro) [reply package.json:158]: That was the note in my slack message, I can add a note here too.

Comment (amphro) [reply package.json:158]: If only you could have commentes in package.json.

**Review by WillieRuemmele — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -142,67 +143,46 @@
...
-    "release-mac-debug": "env SKIP_AWS=true yarn release-mac",
-    "release-standalone": "scripts/release/standalone",
-    "release-standalone-debug": "env SKIP_AWS=true yarn release-standalone",
-    "release-windows": "scripts/release/installers/windows_docker",
-    "release-windows-debug": "env SKIP_AWS=true yarn release-windows",
+    "promote": "oclif promote",
     "snapshot-generate": "HOME=./ ./bin/run snapshot:generate",
-    "test": "scripts/test/all",
-    "test:deprecation-policy": "HOME=./ ./bin/run snapshot:compare",
+    "test": "nyc --report lcov --report cobertura mocha -t 240000 --require source-map-support/register --recursive \"dist/**/*.test.js\"",
```
Comment (mshanemc) [inline package.json:177]: what's gonna be in test/uts ?

Comment (mshanemc) [reply package.json:177]: where do the cobertura stuff go?

**Review by mshanemc — APPROVED** [review]
approved, asked some question, generally mystified

**Review by shetzel — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (amphro) [reply package.json:177]: That is probably legacy stuff too. I wasn't sure so I just left it for now.

**Pull request merged**